src/main/java/core/basesyntax/figuresuppliers/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.figuresuppliers;
+
+import core.basesyntax.color.ColorSupplier;
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        int sizeOne = new Random().nextInt(10) + 1;
+        int sizeTwo = new Random().nextInt(10) + 1;
+        int sizeThree = new Random().nextInt(10) + 1;

```

Comment:
I believe we can improve this and create each side  directly in `case` section. By doing so, we will avoid creation of not needed sides. Also, please do not create new `Random` instance each time you need to create a side value. Instead, as I mentioned, make `Random` a class member and reuse it.